MapboxNavigation/FeedbackItem.swift

```diff
@@ -10,21 +10,20 @@ extension UIImage {
 
 public extension FeedbackType {
 
-    // TODO: Localize these strings
     internal var title: String {
         switch self {
             case .general:
-                return "Feedback"
+                return NSLocalizedString("FEEDBACK_TYPE_GENERAL", bundle: .mapboxNavigation, value: "Feedback", comment: "Feedback type for general or unknown issue")
             case .incorrectVisual(_):
-                return "Incorrect Visual"
+                return NSLocalizedString("FEEDBACK_TYPE_INCORRECT_VISUAL", bundle: .mapboxNavigation, value: "Incorrect Visual", comment: "Feedback type for incorrect visual")
```

Comment:
This is ungrammatical, which will surely hinder some users from choosing this option even when appropriate. It’s also difficult to translate for the same reason. I’d suggest “Looks Incorrect” as an alternative.

/cc @abhishek1508 @asinghal22 @avi-c

Reply:
Updated as @1ec5 suggested.

Reply:
P.S.: @mapbox/navigation-android might want to change this as well: 
https://github.com/mapbox/mapbox-navigation-android/blob/7de359e97901df944a0c53c8f452da2265d1958c/libnavigation-ui/src/main/res/values/strings.xml#L21-L22

Reply:
thanks @1ec5 and @dersim-davaod . I will make changes accordingly on Android